mongodb/README.md

```diff
@@ -1,44 +1,95 @@
 ## Quick Start
 
-This section describes how to run YCSB on MongoDB running locally. 
+This section describes how to run YCSB on MongoDB. 
 
 ### 1. Start MongoDB
 
 First, download MongoDB and start `mongod`. For example, to start MongoDB
 on x86-64 Linux box:
 
-    wget http://fastdl.mongodb.org/linux/mongodb-linux-x86_64-1.8.3.tgz
-    tar xfvz mongodb-linux-x86_64-1.8.3.tgz
+    wget http://fastdl.mongodb.org/linux/mongodb-linux-x86_64-x.x.x.tgz
+    tar xfvz mongodb-linux-x86_64-*.tgz
     mkdir /tmp/mongodb
-    cd mongodb-linux-x86_64-1.8.3
+    cd mongodb-linux-x86_64-*
     ./bin/mongod --dbpath /tmp/mongodb
 
-### 2. Set Up YCSB
+Replace x.x.x above with the latest stable release version for MongoDB.
+See http://docs.mongodb.org/manual/installation/ for installation steps for various operating systems.
 
-Clone the YCSB git repository and compile:
+### 2. Install Java and Maven
```

Comment:
This sounds like a gap in our overall YCSB docs. individual db bindings shouldn't have to tell folks to go get java.

I'll write a follow on issue to deal with this across modules.